autoload/dirvish.vim

```diff
@@ -50,6 +57,17 @@ function! s:globlist(pat) abort
 endfunction
 endif
 
+function! s:get_path(is_relative, key, path) abort
+  let l:full_path = fnamemodify(a:path, ':p')
+  let l:mark = s:get_column_for_path(l:full_path)
+
+  if a:is_relative
+    let l:full_path = fnamemodify(l:full_path, ':.')
+  endif
+
+  return l:mark.l:full_path
```

Comment:
IIUC you are inserting the mark in the buffer. 

Can't conceal be used instead? Probably difficult to get the syntax definition correct. But one of Dirvish's main design goals is to avoid extra junk in the buffer contents. (Whitespace is probably a reasonable exception.)

Reply:
main point of this PR is to avoid conceal because of the difficult syntax definitions. There shouldn't be any issues working with the single or range of files, since the columns are stripped before doing any operations on them.

Reply:
You're right, i didn't think about those.
I'll try to use conceal to achieve this and we will see.